src/main/scala/coupledL2/GrantBuffer.scala

```diff
@@ -31,14 +31,19 @@ import coupledL2.utils.{XSPerfAccumulate, XSPerfHistogram, XSPerfMax}
 class InflightGrantEntry(implicit p: Parameters) extends L2Bundle {
   val set   = UInt(setBits.W)
   val tag   = UInt(tagBits.W)
-  val sink  = UInt(mshrBits.W)
 }
 
 class TaskWithData(implicit p: Parameters) extends L2Bundle {
   val task = new TaskBundle()
   val data = new DSBlock()
```

Comment:
remove TaskWithData

Reply:
TaskWithData should be used in io_d_task